README.md

```diff
@@ -593,7 +593,7 @@ class Comment extends \Spot\Entity
 {
     // ... snip ...
 
-    public static function relations(Mapper $mapper, Entity $entity)
+    public static function relations()
```

Comment:
You can't just remove this code - nothing will work without it. You just need to fix the type-hints:

```php
use Spot\EntityInterface as Entity;
use Spot\MapperInterface as Mapper;

public static function relations(MapperInterface $mapper, EntityInterface $entity)
```